.github/workflows/ci-pull.yml

```diff
@@ -18,10 +18,36 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    #services: TODO add couchdb container for testing
+    services: 
+      cloudant:
+        #Update Readme to match
+        image: couchdb:latest
+        env:
+            COUCHDB_USER: admin
+            COUCHDB_PASSWORD: password
+        # Set health checks to wait until couchdb has started
+        options: >-
+          --health-cmd "curl -f http://localhost:5984/_up || exit 1"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5984:5984
         
 
     steps:
+    # Start CouchDB and initialize required databases
+    - name: Wait for CouchDB to be ready
```

Comment:
Is this step necessary? I thought the health check was used to wait for the container to be ready

Reply:
Not required actually. Initially the build was failing because of unhealthy container. That time I tried to run by giving this and forgot to remove from code. Will remove this commit the code.